src/blog/posts/forward-to-2020.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Looking forward to 2020
+published: true
+description: Moving Heaps Good Dev from PoC to MVP.
+tags: ["post", "meta"]
+slug: forward-to-2020
+date: 2020-02-23
+authors: ["scottcabot"]
```

Comment:
I think we need to add a 'heapsgood' author. That way itll spit out the actual author + this being an 'official' heapsgood blog post. Or we could do it some other way (a boolean frontmatter or something)

Reply:
I have added one now, if this conflicts with your style tweaks (if you added a new profile) we might be able to blow away the one I added but the new blog post includes hgd as the first author.